### PR TITLE
Fix for the failure of etcd shutdown phase

### PR DESCRIPTION
**Pull request by UnderGreen**
## Description

Fixes the issue with etcd shutdown phase during upgrade when etcd server is running inside pod(s).

+ I bumped e reference, which I forgot to do in previous PR for nodejs.

## Type of change

* Bug fix (non-breaking change which fixes an issue)

## Linked tickets and other PRs

* Requires https://github.com/gravitational/planet/pull/715

## TODOs

- [ ] Perform manual testing

### CODE REVIEW

File: Makefile
Comment (a-palchikov) [inline Makefile:49]: Don't forget to change to release before merging.

**Review by a-palchikov — APPROVED** [review]

**Review by r0mant — APPROVED** [review]
lgtm if tests pass (they're failed now)

**Pull request closed without merging**
